tests/test-rest-post-meta-fields.php

```diff
@@ -230,6 +230,37 @@ public function test_set_value() {
 	}
 
 	/**
+	 * @depends test_get_value
+	 */
+	public function test_set_duplicate_single_value() {
```

Comment:
Do we need to test duplicate with multiple, or is that covered in another test / PR?